xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -793,6 +793,9 @@ void CGUIDialogVideoInfo::Play(bool resume)
   // close our dialog
   Close(true);
 
+  // play the current video version, even if multiple versions are available
+  m_movieItem->SetProperty("prohibit_choose_video_version", true);
```

Comment:
Although this is just an arbitrary string I'd change it to the actual cause that will then block the action and not the action itself. You're augmenting the item with a new property which should be independent from the respective action. E.g. if for some reason you kill the choose dialog it does not make sense anymore.
Make it like has_resolved_version or something similar

Reply:
I agree in general , but this property is just a temporary construct and was not introduced by this PR. I don’t want to put too much effort into this interim stuff.

Reply:
Okay, what you say makes sense. I will change the property name in a separate commit. Thanks.

Reply:
done.